pkg/apis/veleroplugin/v1/download.go

```diff
@@ -30,6 +30,13 @@ type DownloadSpec struct {
 	// The server's time is used for SnapshotTimestamp
 	// +optional
 	RestoreTimestamp *meta_v1.Time `json:"restoreTimestamp,omitempty"`
+
+	// ProtectedEntityID is the identifier for the protected entity.
```

Comment:
This is for the volume to overwrite, correct?  Would you add that to the comment?

Reply:
Sure.